sdk/data/azcosmos/go.mod

```diff
@@ -4,9 +4,9 @@ go 1.18
 
 require (
 	github.com/Azure/azure-sdk-for-go v68.0.0+incompatible
-	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.4.0
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.9.2
```

Comment:
Should we do it to https://github.com/Azure/azure-sdk-for-go/issues/22286#issuecomment-1958086028 when that is released?

Reply:
Yes, when 1.10.0 ships, then this should be updated again.